sky/engine/BUILD.gn

```diff
@@ -35,6 +35,10 @@ config("non_test_config") {
 }
 
 group("engine") {
+  public_configs = [
```

Comment:
I think exposing `:config` as public is a bit too broad. It will apply the `cflags` and `include_dirs` to all targets that depend on the `engine`. For example, adding the `-Wno-inconsistent-missing-override` flag is an implementation detail of the way the engine is built and should probably not apply to dependents of the engine.